server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -229,7 +229,7 @@ const memberMutations = {
           });
 
           const policy = getPolicy(host, POLICIES.COLLECTIVE_MINIMUM_ADMINS);
-          if (policy?.numberOfAdmins <= adminCount) {
+          if (policy?.numberOfAdmins && adminCount <= policy.numberOfAdmins) {
```

Comment:
The check was reversed here (`minNumberOfAdmins <= adminCount`). I also added a `policy?.numberOfAdmins` before; in my experience it's safer to separate the nullish condition check because:

```es6
$ undefined < 5
false
$ null < 5
true
```